.github/workflows/release-java-sdk-to-maven-central.yml

```diff
@@ -14,14 +14,9 @@ on:
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  OSSRH_REPO_URL: https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/
   OSSRH_USERNAME: ${{ secrets.OSSRH_USERNAME }}
   OSSRH_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
 
-  # Include only last 8 hex digits of the key ID included, due to
-  # limitations of gradle.
-  SIGNING_KEY_ID: ${{ secrets.SIGNING_KEY_ID }}
```

Comment:
Not used anymore, looks like Gradle should be able to infer from the key material.